osi_sensordata.proto

```diff
@@ -313,4 +313,23 @@ message SensorData
     // in cartesian coordinates.
     //
     optional LogicalDetectionData logical_detection_data = 27;
+
+    //
+    // \brief Virtual detection area of a sensor
+    //
+    // The virtual detection area describes the area the sensor can see in a best case scenario
```

Comment:
I would appreciate some more clarification and this point. Some thoughts to discuss:

1. What about ghost targets from multipath propagation that are actually outside the virtual detection area and appear inside?
2. I am very critical of the term "best case scenario". Is the idea of the virtual coverage area to eventually account for degradation due to environmental effects or not? - I thought it was possible, but the term here suggests the opposite. What do you think about our following definition from the [PerCollECT paper](https://tuprints.ulb.tu-darmstadt.de/18949/1/2021_ITSC_Specify_Sensor_Model_PrePrint.pdf) (p. 2) to use for "best case"? 

> During emission, propagation, reception, and signal processing, the information in the signal deviates from the original information about the environment, existing as ground truth (GT) under clean room conditions. Clean room conditions describe idealized measurement conditions, without the influence of weather, dust, dirt etc. The GT is taken in this clean room environment by an omniscient observer.

Reply:
Would it be a problem if this message represents what the sensor model thinks about its own range in the current time frame under the current conditions? I mean, if the sensor model does not take such conditions into account (for its output/for its detection area), the area may as well stay the same all the time and be very basic. The requirements could vary depending on the use case.

Also, if I think about manufacturer specifications of sensors, they probably do not provide such clean room ranges/FoV's but rather realistic ranges under average conditions. They might even cut off the range by software. For certain use cases such "expected" ranges/FoV's are probably better base lines for visualization for comparison with the corresponding sensor data.

Reply:
Suggestion: The virtual detection area describes a nominal view of the sensor.

Reply:
```suggestion
    // The virtual detection area describes the nominal area the sensor is capable of covering
```